t/39-tinycv.t

```diff
@@ -0,0 +1,372 @@
+        {
+            retval => 2,
+            descr => 'Check 2 for area overlapping top edge.',
+            area => {xpos => 90, ypos => 90, height => 20, width => 120}
+        },
+        {
+            retval => 3,
+            descr => 'Check 3 for area overlapping bottom edge.',
+            area => {xpos => 90, ypos => 190, height => 20, width => 120}
+        },
+        {
+            retval => 4,
+            descr => 'Check 4 for area overlapping left edge.',
+            area => {xpos => 90, ypos => 90, height => 120, width => 20}
+        },
+        {
+            retval => 5,
+            descr => 'Check 5 for area overlapping right edge.',
+            area => {xpos => 190, ypos => 90, height => 120, width => 20}
+        },
+        {
+            retval => 6,
+            descr => 'Check 6 for horizontal split.',
+            area => {xpos => 90, ypos => 140, height => 20, width => 120}
+        },
+        {
+            retval => 7,
+            descr => 'Check 7 for vertical split.',
+            area => {xpos => 140, ypos => 90, height => 120, width => 20}
+        },
+        {
+            retval => 8,
+            descr => 'Check 8 for area overlapping top left corner.',
+            area => {xpos => 90, ypos => 90, height => 20, width => 20}
+        },
+        {
+            retval => 9,
+            descr => 'Check 9 for area overlapping top right corner.',
+            area => {xpos => 190, ypos => 90, height => 20, width => 20}
+        },
+        {
+            retval => 10,
+            descr => 'Check 10 for area overlapping bottom left corner.',
+            area => {xpos => 90, ypos => 190, height => 20, width => 20}
+        },
+        {
+            retval => 11,
+            descr => 'Check 11 for area overlapping bottom right corner.',
+            area => {xpos => 190, ypos => 190, height => 20, width => 20}
+        },
+        {
+            retval => 12,
+            descr => 'Check 12 for partial overlap through top edge.',
+            area => {xpos => 140, ypos => 90, height => 20, width => 20}
+        },
+        {
+            retval => 13,
+            descr => 'Check 13 for partial overlap through bottom edge.',
+            area => {xpos => 140, ypos => 190, height => 20, width => 20}
+        },
+        {
+            retval => 14,
+            descr => 'Check 14 for partial overlap through left edge.',
+            area => {xpos => 90, ypos => 140, height => 20, width => 20}
+        },
+        {
+            retval => 15,
+            descr => 'Check 15 for partial overlap through right edge.',
+            area => {xpos => 190, ypos => 140, height => 20, width => 20}
+        }
+    );
+    my $area1 = {xpos => 100, ypos => 100, width => 100, height => 100};
+
+    foreach my $test_case (@test_case_list) {
+        ok(
+            tinycv::Image::overlap_lvl($area1, $test_case->{area}) eq
+              $test_case->{retval},
+            $test_case->{descr}
+        );
+    }
```

Comment:
```suggestion
    is tinycv::Image::overlap_lvl($area1, $_->{area}), $_->{retval}, $_->{descr}
        for @test_case_list;
```

Reply:
Thanks. I have changed it as you suggested. Saves some space.